doc/users/prev_whats_new/whats_new_3.3.0.rst

```diff
@@ -184,163 +219,418 @@ for out-of-range values with colors that differ from adjacent in-range colors.
+        # only show the bottom spine
+        ax.yaxis.set_major_locator(ticker.NullLocator())
+        for spine in ['top', 'left', 'right']:
+            ax.spines[spine].set_visible(False)
 
-If ``shading='flat'`` then *X*, and *Y* should have dimensions one larger
-than *C*.  If *X* and *Y* have the same dimensions as *C*, then the previous
-behavior is used and the last row and column of *C* are dropped, and a
-DeprecationWarning is emitted.
+        # define tick positions
+        ax.xaxis.set_major_locator(ticker.MultipleLocator(1.00))
+        ax.xaxis.set_minor_locator(ticker.MultipleLocator(0.25))
 
-Users can also specify this by the new :rc:`pcolor.shading` in their
-``.matplotlibrc`` or via `.rcParams`.
+        ax.tick_params(which='major', width=1.00, length=5)
+        ax.tick_params(which='minor', width=0.75, length=2.5, labelsize=10)
+        ax.set_xlim(0, 5)
+        ax.set_ylim(0, 1)
+        ax.text(0.0, 0.2, f'ax.xaxis.set_major_formatter({title})',
+                transform=ax.transAxes, fontsize=14, fontname='Monospace',
+                color='tab:blue')
 
-See :doc:`pcolormesh </gallery/images_contours_and_fields/pcolormesh_grids>`
-for examples.
+        ax.xaxis.set_major_formatter(formatter)
+
+``Axes.set_title`` gains a *y* keyword argument to control auto positioning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`~.axes.Axes.set_title` tries to auto-position the title to avoid any
+decorators on the top x-axis. This is not always desirable so now *y* is an
+explicit keyword argument of `~.axes.Axes.set_title`. It defaults to *None*
+which means to use auto-positioning. If a value is supplied (i.e. the pre-3.0
+default was ``y=1.0``) then auto-positioning is turned off. This can also be
+set with the new rcParameter :rc:`axes.titley`.
+
+.. plot::
 
+    fig, axs = plt.subplots(1, 2, constrained_layout=True, figsize=(5, 2))
+    axs[0].set_title('y=0.7\n$\sum_{j_n} x_j$', y=0.7)
+    axs[1].set_title('y=None\n$\sum_{j_n} x_j$')
+    plt.show()
 
+Offset text is now set to the top when using ``axis.tick_top()``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Solves the issue that the power indicator (e.g., 1e4) stayed on the bottom,
+even if the ticks were on the top.
 
 Set zorder of contour labels
-----------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 `~.axes.Axes.clabel` now accepts a *zorder* keyword argument making it easier
 to set the *zorder* of contour labels. If not specified, the default *zorder*
 of clabels used to always be 3 (i.e. the default *zorder* of `~.text.Text`)
 irrespective of the *zorder* passed to
-`~.axes.Axes.contour`/`~.axes.Axes.contourf`.  The new default *zorder* for
+`~.axes.Axes.contour`/`~.axes.Axes.contourf`. The new default *zorder* for
 clabels has been changed to (``2 + zorder`` passed to `~.axes.Axes.contour` /
 `~.axes.Axes.contourf`).
 
 
+Plot-type specific changes
```

Comment:
Plotting specific changes, but also I am ok w/ lumping 5 things into other (and all 5 are kinda plot/artist modifications...) 🤷

Reply:
Alright, I just put them all together.